Utils/spark-tools/pom.xml

```diff
@@ -19,7 +19,6 @@
     <properties>
         <scala.version.major>2.11</scala.version.major>
         <scala.version.minor>12</scala.version.minor>
-        <scala.version>${scala.version.major}.${scala.version.minor}</scala.version>
```

Comment:
May this change cause build failure.

Reply:
Thank Wei to point out, I dismissed this change that should not be commited